bsvNew.py

```diff
@@ -221,6 +227,10 @@ def main():
     if not args.project_name[0].isupper():
         print("Project name needs to start with a capital letter. Please chose a different name.")
         sys.exit(1)
+    
+    if not is_legal_name(args.project_name):
+        print("Project name needs to NOT have '-' in the name ant NOT be any special keyword. \nPlease chose a different name")
```

Comment:
This message has to be changed to reflect the change with islower.

Reply:
The last case that I put for `valueOf` is already cover by `isupper()`. Should I add to `is_legal_name` the `isupper()` verification case and delete the `if` statement in the line 227?

Reply:
I think it is fine to have another check for isupper to have better error messages.

Otherwise, this looks fine!

Thanks for the contribution.